# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mesa/glibc-2.4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Avoid dprint name-clash with latest glibc.

  - Rene Rebe <rene@exactcode.de>

--- Mesa-6.4.2/src/glu/sgi/libnurbs/interface/mystdio.h.vanilla	2006-06-17 15:09:20.680000000 +0200
+++ Mesa-6.4.2/src/glu/sgi/libnurbs/interface/mystdio.h	2006-06-17 15:13:43.890000000 +0200
@@ -51,12 +51,14 @@
 #include <stdio.h>
 #define dprintf printf
 #else
-inline void dprintf( char *, ... ) { }
+#include <stdio.h>
+#define dprintf mydprintf
+inline void mydprintf( char *, ... ) { }
 #endif
 #endif
 
 #ifdef GLBUILD
-inline void dprintf( char *, ... ) { }
+inline void mydprintf( char *, ... ) { }
 #endif
 
 #ifndef NULL
